Verify_Registration_BQ-77.py

```diff
@@ -15,25 +14,24 @@
 my_account=driver.find_element(By.LINK_TEXT, "My account")
 my_account.click()
 
-    
+#Create random email ids
+letters = string.ascii_letters
+length=10
+result = ''.join(random.choice(letters) for _ in range(length))
+email=result+"@gmail.com"
+print(f"The email created is: {email}")
+
 #Enter the test data to register (later write a program to randomly create email ids so you dont get the "User already exists" error)
 login_email=driver.find_element(By.ID,"reg_email")
-login_email.send_keys("qwerty25@gmail.com")
+login_email.send_keys(email)
```

Comment:
good job using random email. You've figured it out. If you use the same email the second time the script runs the user is already registered so it will fail saying user exists.